install-app.sh

```diff
@@ -13,7 +13,7 @@ apt-get install python3-pip -yq
 # Create a directory for the app and download the files. 
 mkdir /app 
 # make sure to uncomment the line bellow and update the link with your GitHub username
-# git clone https://github.com/<your-gh-username>/azure_task_12_deploy_app_with_vm_extention.git
+git clone https://github.com/p1tonidze/azure_task_12_deploy_app_with_vm_extention.git
```

Comment:
Please update the GitHub repository URL to reflect your own GitHub username instead of 'p1tonidze'. This change is essential for the deployment to work correctly.